src/main/scala/edu/berkeley/ce/rockslicing/RockSlicer.scala

```diff
@@ -29,12 +29,15 @@ object RockSlicer {
     var blocks = Seq(Block(globalOrigin, rockVolume))
 
     // Generate a list of initial blocks before RDD-ifying it
-    val (seedJoints, remainingJoints) = generateSeedJoints(joints, arguments.numSeedJoints)
-    seedJoints foreach { joint =>
-      blocks = blocks.flatMap(_.cut(joint, arguments.minRadius, arguments.maxAspectRatio))
+    if (arguments.numSeedJoints > 0) {
```

Comment:
I'm kind of thinking we can get rid of the seed joints idea and just use processor joints to give a region of the rock volume to each processor. Seems like this would simplify things, since it's one less parameter the user has to understand. What do you think?